garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/material.lua

```diff
@@ -127,9 +127,9 @@ end
 
 function TOOL.BuildCPanel( CPanel )
 
-	CPanel:AddControl( "Header", { Description = "#tool.material.help" } )
+	CPanel:Help( "#tool.material.help" )
 
-	local filter = CPanel:AddControl( "TextBox", { Label = "#spawnmenu.quick_filter_tool" } )
+	local filter = CPanel:TextEntry( "#spawnmenu.quick_filter_tool", nil )
```

Comment:
Surely we can omit `nil` from here, since omitting it would be identical to providing `nil`.